interventions/models.py

```diff
@@ -14,4 +14,11 @@ class Operation(models.Model):
     date = models.DateField()
     time = models.TimeField()
 
+class Patient(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    disease = models.CharField(max_length=128)
+    start_date = models.DateField()
+    main_dr = models.CharField(max_length=32, default="Pole na przypisanie doktora")
```

Comment:
Jeżeli chcemy przypisać doktora to będzie to raczej pole relacji z doktorem. Oczywiście takiego modelu jeszcze nie ma, na razie accept ale dodam odpowiedni issue

Reply:
Z doktorem byłem tej samej myśli że na pózniej